ion/symboltable.go

```diff
@@ -579,3 +579,7 @@ func buildIndex(symbols []string, offset uint64) map[string]uint64 {
 
 	return index
 }
+
+func newString(value string) *string {
+	return &value
+}
```

Comment:
Do we really need this function?

Reply:
I added it because we used it in [symboltable.go. ](https://github.com/amzn/ion-go/pull/158/commits/6418698233b24994b0359aa975cc91a38e74fe9b#diff-574c129c2b9f7fc5906163e00593ba0aR160)

Moved it to symboltable_test.go instead